docs/usage/customization-guide.md

```diff
@@ -278,8 +278,8 @@ trying to execute them. A subdirectory under the hooks directory can be used,
 for example `/etc/kubernetes/node-feature-discovery/source.d/conf/`.
 
 **NOTE:** Hooks are being DEPRECATED and will be removed in a future release.
-For backward compatibility, currently hooks are enabled by default and can be
-disabled via `sources.local.hooksEnabled` field in the worker configuration.
+Starting from release v0.13.2 hooks are disabled by default and can be
```

Comment:
no objections for this PR but I find it surprising this change is done in the middle of `v0.13` cycle

Reply:
Agree, please move to v0.14, the final deprecation will happen N+2 (v0.16)

Reply:
Yeah, no earlier than v0.14. We definitely don't want to break existing users with a patch release

Reply:
agreed, will move to v0.14